pages/CreatingAnExploration.html

```diff
@@ -9,20 +9,22 @@
 
   <p>
     After you've played an exploration, why not try your hand at making one? To
-    do so, navigate to the gallery, and look for the "Create Exploration"
+    do so, navigate to the gallery, and look for the "Create"
     button in the top navbar.
   </p>
 
-  <img src="static/images/galleryCreateButton.png" style="width: 100%;">
+  <img src="static/images/galleryCreateButton2.png" style="width: 100%;">
 
   <p>
-    This will open a dialog box.
+    This will take you to a new page, click on the setting icon on the top right 
```

Comment:
This will take you to the exploration editor page. Click on the Settings icon at the top right of the page; this will take you to a page where you can fill in the basic settings for your exploration.